src/jwt.cpp

```diff
@@ -28,31 +28,37 @@ static const std::string base64_chars =
     "abcdefghijklmnopqrstuvwxyz"
     "0123456789+/";
 
-std::string base64_encode(const unsigned char* bytes_to_encode,
-                          unsigned int in_len) {
+std::string base64_encode(const unsigned char *bytes_to_encode,
```

Comment:
Prefer String (native Arduino type) over std::string as per Arduino style. You can pull in from my branch to get these updates.